core/blockchain_snapshot_test.go

```diff
@@ -348,11 +349,13 @@ func (snaptest *setHeadSnapshotTest) test(t *testing.T) {
 // - commit the snapshot
 // - crash
 // - restart again
+//nolint:unused
 type restartCrashSnapshotTest struct {
 	snapshotTestBasic
 	newBlocks int
 }
 
+//nolint:unused
```

Comment:
@rjl493456442 this comes from https://github.com/ethereum/go-ethereum/pull/22163/files#diff-3ecc1958cb0ed978a0edcf6c43089b130034057420e4ce2e1e0c447cd7cec454R968 -- it's not used, it was originally used in `TestRecoverSnapshotFromCrashWithLegacyDiffJournal` (which was Skipped) but not is no longer in the codebase. It seems to have been removed here: https://github.com/ethereum/go-ethereum/commit/d6ffa140359d24262903962f145a1236836aeeb4. 

So I'll just remove these structs then.. ?

Reply:
It's ok to remove